feature-tests/utils/send-sqs-message.ts

```diff
@@ -0,0 +1,26 @@
+import { fromSSO } from '@aws-sdk/credential-providers';
+import { SQS } from '@aws-sdk/client-sqs';
+import { aisEvents } from './ais-events-array';
+
+export async function sendSQSEvent(testUserId: string, aisEventType: keyof typeof aisEvents) {
+  const credentials = await fromSSO({
+    profile: process.env['AWS_PROFILE'] ?? 'dev',
+  })();
+
+  const sqs = new SQS({ apiVersion: '2012-11-05', credentials, region: 'eu-west-2' });
```

Comment:
I think this would be a lot more flexible/easy to use if this just took the event as a json object

Reply:
Hi Chiara, will raise this with you in the next Dev Techie and see how we can implement the change on the next PR for feature-tests, thank you